theforeman.org/scripts/test/test_develop.sh

```diff
@@ -32,11 +32,11 @@ bundle exec rake rubocop
 
 # Database environment
 (
```

Comment:
can you replace `gemset` with something like `jobid` and do before this block
```
jobid=$(echo ${JOB_NAME} | cut -d/ -f1)-${EXECUTOR_NUMBER}
```

the databases need to be unique